tests/api2/test_273_kubernetes.py

```diff
@@ -20,6 +20,7 @@ def test_01_get_kubernetes_bindip_choices():
     assert results.status_code == 200, results.text
     assert isinstance(results.json(), dict), results.text
     assert results.json()['0.0.0.0'], results.text
+    assert results.json()[ip], results.text
```

Comment:
If ip does not exist, we will have a keyerror. I think we should do “ip in results.json()” to check if it exists

Reply:
good idea

Reply:
I will do the same for 0.0.0.0

Reply:
Done